README.md

```diff
@@ -2432,6 +2432,37 @@ config.properties['hazelcast.client.statistics.period.seconds'] = 4;
 
 After enabling the client statistics, you can monitor your clients using Hazelcast Management Center. Please refer to the [Monitoring Clients section](https://docs.hazelcast.org/docs/management-center/latest/manual/html/index.html#monitoring-clients) in the Hazelcast Management Center Reference Manual for more information on the client statistics.
 
+### 7.9.2. Logging Configuration
+
+Hazelcast Node.js client has a flexible logging configuration. You can add your custom loggers by providing adapters.
+
+To configure a logger, you need to implement the `ILogger` interface in your adapter and add it to `ClientConfig.properties['hazelcast.logging']`. If you set this property to `'off'`, it does not log anything.
+
+See the following built-in `winston` logging library example.
+
+```javascript
+var winstonAdapter = {
+    logger: new (winston.Logger)({
+        transports: [
+            new (winston.transports.Console)()
+        ]
+    }),
+
+    levels: [
+        'error',
+        'warn',
+        'info',
+        'debug',
+        'silly'
+    ],
+
+    log: function (level, className, message, furtherInfo) {
+        this.logger.log(this.levels[level], className + ' ' + message);
+    }
+};
+config.properties['hazelcast.logging'] = winstonAdapter;
+```
+
```

Comment:
AFAIK it is not possible to configure custom logging via declarative configuration. It is good to state this too. Otherwise, users may waste time trying to find a configuration that does not exist.